_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Custom Clock Processes
+nav: Custom Clock Processes
+modified_at: 2021-10-19 10:00:00
+tags: task-scheduling
+index: 2
+---
+
+With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
+environment parity between development and production, or time precision in your task scheduling.
+
+## Definition of the Custom Clock
+
+The implementation of the custom clock will vary depending on the used language. However, defining the clock process
+is very simple and is done in the `Procfile`.
+
+#### Example 
+
+Here is an example of the process definition using the Ruby library`clockwork`:
+```yaml
+web: bundle exec puma -t 1:3 -p $PORT
```

Comment:
nitpick: I would use the same puma command as we already do in the doc (https://doc.scalingo.com/languages/ruby/start#using-puma-as-a-production-server)

```suggestion
web: bundle exec puma -C config/puma.rb
```